### ISSUE TITLE
Figure referencing: markdown syntax

### ISSUE THREAD

**Issue by rossbar** [rossbar/elegant-scipy#9]
Figure referencing seems to require rst syntax in ipypublish. For example:

```
I am referring to @fig:example
```
Does not work whereas
```tex
I am referring to \:numref\:\`fig:example\`
```
does.

**Comment by rossbar** [comment]
The filters in the [pandoc-fignos](https://github.com/tomduck/pandoc-fignos) package are designed to solve this problem. Tried simply installing to see if anything happened but it appears ipypublish is not set up to use these filters. Potential for improvement there.

Keep an eye out for this when looking through ipypub docs

**Comment by rossbar** [comment]
Found in [ipypub docs](https://ipypublish.readthedocs.io/en/latest/markdown_cells.html#id15) . Closed by 1a414f4
